### ISSUE TITLE
500 internal server error on POST /vote/ballot

### ISSUE THREAD

**Issue by marcschwaiger** [LeoTuet/backend-we-want-to-sleep#17]
Posting to /vote/ballot on the productive system using
`curl -v https://wewanttosleep.de/api/vote/ballot  -H "Content-Type: application/json" -d '{"token": "token", "vote": "vote"}'`
I get following output: 
```
*   Trying 78.46.198.245:443...
* Connected to wewanttosleep.de (78.46.198.245) port 443 (#0)
* schannel: disabled automatic use of client certificate
* schannel: ALPN, offering http/1.1
* schannel: ALPN, server accepted to use http/1.1
> POST /api/vote/ballot HTTP/1.1
> Host: wewanttosleep.de
> User-Agent: curl/7.79.1
> Accept: */*
> Content-Type: application/json
> Content-Length: 29
>
* Mark bundle as not supporting multiuse
< HTTP/1.1 500 Internal Server Error
< Server: nginx/1.20.2
< Date: Wed, 09 Feb 2022 11:45:42 GMT
< Content-Type: application/json; charset=utf-8
< Content-Length: 12
< Connection: keep-alive
< X-Powered-By: Express
< ETag: W/"c-3Rk1bge0s6VuJbi+S2m2iU0UGdY"
< Vary: Accept-Encoding
<
{"error":{}}* Connection #0 to host wewanttosleep.de left intact
```

expected would be error '401 Token not found' called [here](https://github.com/LeoTuet/backend-we-want-to-sleep/blob/main/src/handler/VoteHandler.ts#L14), instead I get an empty error response with code 500 internal server error

**Comment by 94726** [comment]
Whaaa

**Comment by marcschwaiger** [comment]
With the new implemented input validation
`curl -v https://wewanttosleep.de/api/vote/ballot -H "Content-Type: application/json" -d '{"token": "1111-BBBB-2222", "vote": "vote"}'` 
now produces this error

**Comment by 94726** [comment]
Me and @marcschwaiger were able to find out that the issue happends in the votecontroller with the x-captcha header as it is indexed with uppercase letters and Express lowercases all headers

**Comment by marcschwaiger** [comment]
Changing X-Captcha to x-captcha [here](https://github.com/LeoTuet/backend-we-want-to-sleep/blob/main/src/controller/VoteController.ts#L39) fixes the problem

**Comment by LeoTuet** [comment]
fixed with pull request #27
